Update Readme

Can you update the README to make it clear that you still need to declare

```
androidTestImplementation 'com.jraska:falcon:2.0.1'
```

even if you are using 

```
androidTestImplementation 'com.jraska:falcon-spoon-compat:2.0.1'
```

I initially didn't include the `falcon` dependency since I assumed that it would be pulled in as part of `falcon-spoon-compat`. But when I did I got the following runtime error:

```
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/jraska/falcon/Falcon;
    at com.jraska.falcon.FalconSpoonRule.screenshot(FalconSpoonRule.java:42)
...
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.jraska.falcon.Falcon" on path: DexPathList[[...]]
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ android:
     - tools
     - platform-tools
     - tools
-    - build-tools-27.0.2
+    - build-tools-27.0.3
     - android-27
 
     - extra-google-m2repository

--- a/README.md
+++ b/README.md
@@ -48,6 +48,7 @@ public void awesomeTest() {
 ```
 #### Gradle
 ```groovy
+androidTestCompile 'com.jraska:falcon:2.0.1'
 androidTestCompile 'com.jraska:falcon-spoon-compat:2.0.1'
 ```
 

--- a/build.gradle
+++ b/build.gradle
@@ -6,7 +6,7 @@ buildscript {
     google()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.0.1'
+    classpath 'com.android.tools.build:gradle:3.1.3'
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7'
     classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
   }
@@ -26,7 +26,7 @@ ext {
   minSdkVersion = 14
   compileSdkVersion = 27
   targetSdkVersion = compileSdkVersion
-  buildToolsVersion = '27.0.2'
+  buildToolsVersion = '27.0.3'
   sourceCompatibility = JavaVersion.VERSION_1_7
 }
 

--- a/falcon-sample/build.gradle
+++ b/falcon-sample/build.gradle
@@ -63,8 +63,8 @@ android {
 dependencies {
   implementation project(':falcon')
 
-  implementation 'com.android.support:appcompat-v7:27.0.2'
-  implementation 'com.android.support:design:27.0.2'
+  implementation 'com.android.support:appcompat-v7:27.1.1'
+  implementation 'com.android.support:design:27.1.1'
   implementation 'com.jakewharton:butterknife:8.8.1'
   annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
 
@@ -72,10 +72,10 @@ dependencies {
   androidTestImplementation project(':falcon-spoon-compat')
   //noinspection GradleDependency - 3 cannot be compiled due to JVM 8 instructions
   androidTestImplementation 'org.assertj:assertj-core:2.9.0'
-  androidTestImplementation 'com.android.support.test:runner:1.0.1'
-  androidTestImplementation 'com.android.support.test:rules:1.0.1'
-  androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+  androidTestImplementation 'com.android.support.test:runner:1.0.2'
+  androidTestImplementation 'com.android.support.test:rules:1.0.2'
+  androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 
   // separately set because of test runners older dependencies
-  androidTestImplementation 'com.android.support:support-annotations:27.0.2'
+  androidTestImplementation 'com.android.support:support-annotations:27.1.1'
 }

--- a/falcon-spoon-compat/build.gradle
+++ b/falcon-spoon-compat/build.gradle
@@ -29,7 +29,7 @@ android {
 }
 
 dependencies {
-  implementation 'com.android.support.test:rules:1.0.1'
+  implementation 'com.android.support.test:rules:1.0.2'
   implementation project(':falcon')
   api "com.squareup.spoon:spoon-client:2.0.0-SNAPSHOT"
   api 'junit:junit:4.12'

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
-#Tue Dec 05 16:54:52 CET 2017
+#Tue Jun 26 23:14:20 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip

